src/badge/badge.scss

```diff
@@ -11,14 +11,17 @@ $iui-badge-background-primary: $iui-color-foreground-primary--dark;
 $iui-badge-text-color: rgba($iui-color-foreground-body--light, $iui-opacity-3--light);
 
 @mixin iui-badge {
+  --_iui-badge-background-color: #{$iui-badge-background-default};
+  --_iui-badge-text-color: #{$iui-badge-text-color};
```

Comment:
I have to keep the `$iui-badge-text-color` on line 11 because Sass is able to use hex values as the first parameter of `rgba` but CSS variables cant do that directly in the browser.